### ISSUE TITLE
ModelCube full_fit

### ISSUE THREAD

**Issue by bretthandrews** [sdss/marvin#548]
**Describe the bug**
In [18]: mc.full_fit
---------------------------------------------------------------------------
MarvinError                               Traceback (most recent call last)
<ipython-input-18-4c9617025f20> in <module>()
----> 1 mc.full_fit

/Users/andrewsb/software/miniconda3/envs/marvindev/lib/python3.5/site-packages/marvin/tools/modelcube.py in full_fit(self)
    585                         mask=model_mask,
    586                         redcorr=self._redcorr,
--> 587                         binid=self.get_binid(model),
    588                         unit=model.unit,
    589                         pixmask_flag=model.pixmask_flag)

/Users/andrewsb/software/miniconda3/envs/marvindev/lib/python3.5/site-packages/marvin/tools/modelcube.py in get_binid(self, model)
    543             if response.results['error'] is not None:
    544                 raise MarvinError('found a problem while getting the binid from API: {}'
--> 545                                   .format(str(response.results['error'])))
    546
    547             binid_map_data = np.array(response.getData()['binid'])

MarvinError: found a problem while getting the binid from API: cannot make a list of Quantities.  Get list of values with q.value.list().
You can submit this error to Marvin GitHub Issues (https://github.com/sdss/marvin/issues/new).
Fill out a subject and some text describing the error that just occurred.
If able, copy and paste the full traceback information into the issue as well.

**To Reproduce**
```
from marvin.tools import ModelCube
mc = ModelCube('8241-3704')
mc.full_fit
```

**Comment by albireox** [comment]
What's the configuration for the "To Reproduce". Is this DB, file, API? What MPL? I'm not able to reproduce it.

**Comment by havok2063** [comment]
I think everything is being tested using the `marvindev` environment file from the vetting.  This is the `2.3.0dev` pip release, based off `vetround2` branch.

**Comment by bretthandrews** [comment]
Sorry I created this issue during the crash day and forgot to go back and flesh it out, but Brian covered the important bits. I believe that I was trying to use the API with probably DR15 but maybe MPL-7.

**Comment by albireox** [comment]
There was a problem converting the binid extension from a `Map` (and thus a quantity) to a list. Fixed in f344343fd06bbbdda7f08da6fb2777611f4f5d5c
